node/pkg/processor/broadcast.go

```diff
@@ -31,26 +31,46 @@ func (p *Processor) broadcastSignature(
 	txhash []byte,
 	digest ethCommon.Hash,
 	signature []byte,
-) (*gossipv1.SignedObservation, []byte) {
-	obsv := gossipv1.SignedObservation{
-		Addr:      p.ourAddr.Bytes(),
+	shouldPublishImmediately bool,
+) (ourObs *gossipv1.Observation, msg []byte) {
+	// Create the observation to either be submitted to the batch processor or published immediately.
+	ourObs = &gossipv1.Observation{
 		Hash:      digest.Bytes(),
 		Signature: signature,
 		TxHash:    txhash,
 		MessageId: messageID,
 	}
 
-	w := gossipv1.GossipMessage{Message: &gossipv1.GossipMessage_SignedObservation{SignedObservation: &obsv}}
+	if batchCutoverComplete() {
+		if shouldPublishImmediately {
+			msg = p.publishImmediately(ourObs)
+			observationsBroadcast.Inc()
+		} else {
+			p.postObservationToBatch(ourObs)
```

Comment:
Is there a corresponding prometheus metric being pegged for observation batches?

Reply:
Added one.